gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -107,14 +117,54 @@ func (o *options) validate() error {
 	if _, err := os.Stat(o.flStyles); os.IsNotExist(err) {
 		return fmt.Errorf("styles path '%s' doesn't exists.", o.flStyles)
 	}
+	if o.oauthTokenFile != "" && o.gcsCredentialsFile != "" {
+		return errors.New("specifying both --oauth-token-file and --gcs-credentials-file is not allowed.")
+	}
+
+	resolveGCSBaseURL = func() string { return gcsBaseURL }
```

Comment:
well that was in an older commit. It doesn't exist now, since I made the storage client to get the files and output to the screen, we don't need any of the google URLs